nncf/tensor/tensor.py

```diff
@@ -205,6 +205,25 @@ def clone(self) -> Tensor:
     def as_numpy_tensor(self) -> Tensor:
         return cast(Tensor, _call_function("as_numpy_tensor", self))
 
+    def as_openvino_tensor(self) -> Tensor:
+        x = self
+        if x.backend == TensorBackend.torch:
+            x = cast(Tensor, _call_function("as_numpy_tensor", x))
+        if x.backend == TensorBackend.numpy:
+            try:
+                import openvino as ov  # type: ignore
+            except ImportError:
+                msg = "OpenVINO is not installed. Please install OpenVINO to cast a tensor to OpenVINO."
+                raise ImportError(msg)
+
+            from nncf.tensor.functions.openvino_numeric import DTYPE_MAP
+
+            x = Tensor(ov.Tensor(x.data, x.data.shape, DTYPE_MAP[x.dtype]))
```

Comment:
I would suggest don't add support for PyTorch because the implementation is not optimal.

My suggestion for numpy backend:
```suggestion
        if x.backend == TensorBackend.numpy:
            x = cast(Tensor, _call_function("from_numpy", x.data, BackendType.OPENVINO))
```

Reply:
Got it, done.